doc/cpd/101.rst

```diff
@@ -75,6 +78,16 @@ if set to ``True`` would use the secrets handler.
 The secrets handler would be automatically added to the ``StorageHandlerManager`` and
 would require no special configuration by the operator.
 
+New HTTP Handler
+~~~~~~~~~~~~~~~~
+A new handler called ``secrets`` could be added. This would proxy requests back to the
+Custodia instance.
```

Comment:
Okay, so if the "secrets" handler is using a Unix socket to proxy to Custodia, does that imply Custodia has to be running on the same machine as `commissaire-server`?  In the example below, it looks like `commissaire-storage-service` has the Custodia config, and that service may be running on a different machine than `commissaire-server`.

Would it make sense for `commissaire-storage-service` to call back to our own `/api/v0/secrets` endpoint, instead of accessing Custodia directly?

I feel like I'm not seeing a piece of this clearly.